addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-sarif-json.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>Traditional JSON Report with Requests and Responses</TITLE>
```

Comment:
Needs to be updated.

Reply:
@thc202 : I did change this in https://github.com/zaproxy/zap-extensions/pull/4078/commits/eefc4d7610b8653301dc8cade47646b5977af458